### PR TITLE
Fix #2602 : Use string from strings.xml in MyDownloadFragment

### LINKED ISSUE #2602: Use string from strings.xml file in MyDownloadFragmentTest
Below mentioned line of code is using hard-coded string instead this string should be referenced from `strings.xml` file

https://github.com/oppia/oppia-android/blob/ffacea694d5876ed0dd3fae12d57ef68f3e3efb7/app/src/sharedTest/java/org/oppia/android/app/mydownloads/MyDownloadsFragmentTest.kt#L88

### PR DESCRIPTION
**Pull request by kartikeysaran**
## Explanation
Fixes #2602
Previously hard-coded string was used in MyDownloadsFragment , This Pr is using string from strings.xml file

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@kartikeysaran Please comment on the issue first that you are working on it.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

File: .idea/misc.xml
Comment (FareesHussain) [inline .idea/misc.xml]: Revert this change too

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@kartikeysaran the changes before @rt4914 reviewed were enough.

**Review by FareesHussain — APPROVED** [review]

File: .idea/misc.xml
Comment (rt4914) [reply .idea/misc.xml]: @kartikeysaran always make sure that you reply to comments.
In this cane it cane be `Done`

Comment (kartikeysaran) [reply .idea/misc.xml]: @rt4914 okay

Comment (kartikeysaran) [reply .idea/misc.xml]: I have reverted back the changes @FareesHussain thankyou for your help !

**Comment by rt4914** [comment]
@kartikeysaran Please always mention the reason why you are closing the PR. Also, I had approved this PR twice but just after approval you committed some other changes, what was the reason behind that?

**Comment by kartikeysaran** [comment]
> @kartikeysaran Please always mention the reason why you are closing the PR. Also, I had approved this PR twice but just after approval you committed some other changes, what was the reason behind that?

By mistake I committed changes in the PR that was not referring to this issue and the PR was messed up so I build another PR for this issue #2605 and ended this PR . I will make sure before closing any PR from now on mentioning the reason behind it

**Pull request closed without merging**
